src/index.html

```diff
@@ -9,10 +9,48 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container_calendar">
+      <div class="calendar calendar--start-day--sun calendar--month-length--31">
+        <div class="calendar__days"></div>
```

Comment:
```suggestion
        <div class="calendar__day"></div>
```
it is reason of such error.
class names should be exactly as in task description.
<img width="1510" alt="Screenshot 2024-07-08 at 07 56 40" src="https://github.com/mate-academy/layout_calendar/assets/33029864/2b060cb0-7d10-48de-9a7a-ad08e0ddd1a6">